### PR TITLE
Docs update 1.2.7

### PR DESCRIPTION
**Pull request by zeke**
https://github.com/electron/electron/releases/tag/v1.2.7 is here!

### CODE REVIEW

**Comment by jlord** [comment]
screen shot 2016-07-13 at 9 29 11 am

Ooop, lemme look into ☝️

File: _data/processes.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@
 "globalShortcut": "Main Process"
 "ipcMain": "Main Process"
 "ipcRenderer": "Renderer Process"
+"Locales": "Main Process"
```
Comment (kevinsawicki) [inline _data/processes.yml]: I think the ` "Possible Values of app.getLocale()": "Main Process"` line can now be removed from this file.

Comment (zeke) [inline _data/processes.yml]: Done.

**Comment by zeke** [comment]
Okay, I think this is ready now.

**Pull request merged**